README.md

```diff
@@ -14,19 +14,18 @@ MixedRealityToolkit-Unity uses code from the base [MixedRealityToolkit](https://
 > Learn more about [Windows Mixed Reality](https://www.microsoft.com/en-gb/windows/windows-mixed-reality) here
 
 [github-rel]:                  https://github.com/Microsoft/MixedRealityToolkit-Unity/releases/latest
-[github-rel-badge]:            https://img.shields.io/github/tag/microsoft/MixedRealityToolkit-unity.svg?style=flat-square&label=latest%20master%20branch%20release&colorB=007ec6
+[github-rel-badge]:            https://img.shields.io/badge/WMR%20MRTK%20Release-%20v1.Dev.2017.2.1-green.svg
 [![Github Release][github-rel-badge]][github-rel]
 
-[unity-download]:                 http://beta.unity3d.com/download/edcd66fb22ae/download.html
-[unity-version-badge]:            https://img.shields.io/badge/current%20unity%20editor%20version-2017.2.0f3%20MRTP-green.svg
+[unity-download]:                 http://beta.unity3d.com/download/00283454d7e3/download.html
+[unity-version-badge]:            https://img.shields.io/badge/Current%20Unity%20Editor%20Version-2017.2.0f3%20MRTP3-green.svg
 [![Github Release][unity-version-badge]][unity-download]
 
 [githubhl-rel]:                  https://github.com/Microsoft/MixedRealityToolkit-Unity/releases/latest
-[mrtk-version-badge]:            https://img.shields.io/badge/HoloLens%20MRTK%20release-2017.1.2-blue.svg
+[mrtk-version-badge]:            https://img.shields.io/badge/HoloLens%20MRTK%20Release-%20v1.2017.2.0-blue.svg
 
-[unity-release]:
-https://unity3d.com/unity/whats-new/unity-2017.1.2
-[githubhl-rel-badge]:            https://img.shields.io/badge/Unity%20HoloLens%20release-2017.1.2-blue.svg
+[unity-release]:                 https://unity3d.com/unity/whats-new/unity-2017.1.2
+[githubhl-rel-badge]:            https://img.shields.io/badge/Unity%20Editor%20HoloLens%20Release-2017.1.2.0f3-blue.svg
```

Comment:
Why is the [v1.2017.2.0](https://github.com/Microsoft/MixedRealityToolkit-Unity/releases/tag/v1.2017.2.0) release marked for use with Unity 2017.1.2? This should be v1.2017.1.3 instead (or v1.2017.1.2.1 if we really want that many numbers haha).

v1.2017.2.0 should work with Unity 2017.2, per the [roadmap](https://github.com/Microsoft/MixedRealityToolkit-Unity#roadmaps).

Reply:
Generally I try to keep the release tags as close to Unity's as possible.

I actually had matching the release but had to backtrack when we found out that the latest 2017.2 is incompatible with HoloLens building using the later SDK targets.